src/components/LearnWith/GuideNotes.jsx

```diff
@@ -1,66 +1,66 @@
-import {ChatAltIcon, HandIcon} from '@heroicons/react/outline';
+import { Image } from "next/image";
+import { ChatAltIcon } from "@heroicons/react/outline";
 
 const people = [
   {
-    name: 'Leslie Alexander',
-    role: 'Gitcoin Community Manager',
+    name: "Leslie Alexander",
+    role: "Gitcoin Community Manager",
     imageUrl:
-      'https://images.unsplash.com/photo-1494790108377-be9c29b29330?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=facearea&facepad=8&w=1024&h=1024&q=80',
-    bio: 'Ultricies massa malesuada viverra cras lobortis. Tempor orci hac ligula dapibus mauris sit ut eu. Eget turpis urna maecenas cras. Nisl dictum.',
-    twitterUrl: '#',
-    linkedinUrl: '#',
+      "https://images.unsplash.com/photo-1494790108377-be9c29b29330?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=facearea&facepad=8&w=1024&h=1024&q=80",
+    bio: "Ultricies massa malesuada viverra cras lobortis. Tempor orci hac ligula dapibus mauris sit ut eu. Eget turpis urna maecenas cras. Nisl dictum.",
+    twitterUrl: "#",
+    linkedinUrl: "#",
   },
   // More people...
 ];
 
-const GuideNotes = ({notes}) => {
+const GuideNotes = ({ guideNoteData }) => {
   return (
     <div className="mx-auto max-w-7xl">
       <div className="space-y-12 lg:gap-8">
-        <h2 className="text-3xl font-extrabold tracking-tight">
-          Notes from your guide
-        </h2>
+        <h2 className="text-3xl font-extrabold tracking-tight">Guide notes</h2>
         <div className="lg:col-span-2">
           <ul
             role="list"
             className="space-y-12 sm:divide-y sm:divide-gray-200 sm:space-y-0 sm:-mt-6 lg:gap-x-8 lg:space-y-0"
           >
-            {people.map((person) => (
-              <li key={person.name}>
-                <div className="sm:grid sm:grid-cols-4 sm:items-start sm:gap-6 sm:space-y-0">
-                  <div className="image-container sm:col-span-1 space-y-4 max-w-xs">
-                    <div className="aspect-w-3 aspect-h-2 sm:aspect-w-3 sm:aspect-h-4">
+            <li>
+              <div className="space-y-2">
+                <div className="image-container space-x-4 mb-4 max-w-xs flex">
+                  <div className="avatar">
+                    <div className="rounded-full w-20 h-20">
                       <img
-                        className="object-cover shadow-lg rounded-lg"
-                        src={person.imageUrl}
-                        alt=""
+                        src={guideNoteData.user.avatar}
```

Comment:
Can we add a null value check for "user" and other related data on guideNoteData variable like the following?

**ex: guideNoteData.user?.avatar**

Reply:
Yes we can. I should actually abstract this out to a component with all the right checks.

I didn't know about the '?' after the thing you want to check exists. This is handy :)

Could we approve this one and I'll create an issue to use Avatar component where there are user avatars and ensure that the component has the right checks?

Reply:
Created issue for this #53